projects/pastanaga-angular/src/lib/table/table-cell-header-sortable/table-header-button.component.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../styles/variables';
+
+button {
+    color: $color-text-table-header;
+    font-size: font-size(xs);
+    display: flex;
+    align-items: center;
+    border: transparent;
+    background: transparent;
+    padding: 0 rhythm(1);
+    margin: 0 #{- rhythm(1)};
+    font-weight: 700;
+    text-transform: uppercase;
+
+    &:active {
+        color: $color-neutral-primary-default;
+    }
+
+    @media (min-width: $size-viewport-large-min) {
+        pa-icon {
+            opacity: 0;
+        }
+
+        &.active pa-icon {
+            opacity: 1;
```

Comment:
add `transition: transform ease-in-out .3s` so header button feels a bit more animated when changing state. You can probably do the same for `opacity` by the way.